### DISCUSSION TITLE
Can we implement switching account functionality?

### DISCUSSION THREAD

**Discussion by heauton** [supabase/supabase#3402] [Questions]
Is it possible to implement the "switching account" feature with Supabase, where a person can log into different accounts, i.e., one email address per account? If so, could anyone generously provide some guidance? It seems to be more common nowadays, e.g., Twitter, Slack, Instagram, Figma, etc.

**Comment by awalias** [comment]
the accounts are currently linked according to primary email address so if they are the same from 2 given providers, it wouldn't be practical to un-link them

however, we are [about to change this behavior](https://github.com/supabase/gotrue/pull/208) by no longer using the email as the identifier, in which case the behavior you desire will become possible. @kangmingtay may have more details on this one

**Reply by heauton** [reply]
That's good to hear. @kangmingtay  Is the linked PR going to allow multiple identities from the same provider? For example, sign in with _multiple_ google accounts or email addresses?
